torch/testing/_internal/common_methods_invocations.py

```diff
@@ -476,6 +476,8 @@ def method_tests():
         ('expand_as', (S, 1, 1), (torch.rand(S, S, S),), '', (False,)),
         ('exp', (S, S, S), NO_ARGS, '', (True,)),
         ('exp', (), NO_ARGS, 'scalar', (True,)),
+        ('exp2', (S, S, S), NO_ARGS, '', (True,)),
```

Comment:
You can leave these entries for now. Although they'll be mostly redundant with the OpInfo entry there's still a couple tests that only use method_tests.